java/kafka/deployment-tracing.yaml

```diff
@@ -56,6 +56,20 @@ spec:
             value: const
           - name: JAEGER_SAMPLER_PARAM
             value: "1"
+          - name: OTEL_SERVICE_NAME
+            value: kafka-opentelemetry
+          - name: OTEL_TRACES_EXPORTER
+            value: jaeger
+          - name: OTEL_METRICS_EXPORTER
+            value: none
+          - name: OTEL_EXPORTER_JAEGER_ENDPOINT
+            value: http://my-jaeger-collector:14250
+          - name: OTEL_LOG_LEVEL
+            value: debug
+          - name: TRACING_SYSTEM
+            #  value: "opentracing"
```

Comment:
I would remove this comment here, we will default to opentelemetry unless one of the maintainer/reviewers objects